test/e2e/e2e_suite_test.go

```diff
@@ -184,7 +184,7 @@ var _ = Describe("operator-sdk", func() {
 			By("building the operator bundle image")
 			// Use the existing image tag but with a "-bundle" suffix.
 			imageSplit := strings.SplitN(tc.ImageName, ":", 2)
-			bundleImage := path.Join("quay.io", imageSplit[0]+"-bundle")
+			bundleImage := imageSplit[0] + "-bundle"
 			if len(imageSplit) == 2 {
```

Comment:
Hi @joelanford, 
Since you are cleanup it. All these codes could be simplified with `fmt.Sprintf` such as

>  `fmt.Sprintf("e2e-test/controller-manager-bundle:%", tc.testSuffix),`

Reply:
It would be better to avoid assuming image names, although they aren't for the test itself.

Reply:
I think it's better to keep using the existing variables. If `tc.ImageName` ever changed, your suggested change would break, IIUC.

Reply:
tc.ImageName == e2e-test/controller-manager-bundle